### PR TITLE
Minor fixes + Readme.in + Scripts for testing the documentation + Readme update

### PR DESCRIPTION
**Pull request by jmmartinez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,177 @@
...
+About
+-----
+
+Easy::jit is a _compiler-assisted_ library that enables simple just-in-time 
+code generation for C++ codes. 
+
+Building
+--------
+
+First, install clang and the llvm.
```
Comment (serge-sans-paille) [inline README.md]: probably ``and LLVM`` (without *the*)

File: README.md [hunk 2]
```diff
@@ -1,24 +1,177 @@
...
+
+```bash
+apt-get install -y llvm-5.0-dev llvm-5.0-tools clang-5.0
```
Comment (serge-sans-paille) [inline README.md]: in 2018, it's just ``apt install``

File: README.md [hunk 3]
```diff
@@ -1,24 +1,177 @@
...
+-----------
+
+### Compiling my project with Easy::Jit
+
+Since the Easy::Jit library relies on assistance from the compiler, its 
+mandatory to load a compiler plugin in order to use it. 
+The flag ```-Xclang -load -Xclang <path_to_easy_jit_build>/bin/EasyJitPass.so``` 
+loads the plugin.
+
+The included header require C++14 support, and remember to add the include directories!
```
Comment (serge-sans-paille) [inline README.md]: requires

File: README.md [hunk 4]
```diff
@@ -1,24 +1,177 @@
...
+  -L<path_to_easy_jit_build>/bin -lEasyJitRuntime 
+```
+
+### Using Easy::Jit inside my project
+
+Consider the code bellow from a software that applies image filters on a video stream.  
+In the following sections we are going to adapt it to use the Easy::jit library.
+The function to optimize is ```kernel```, which applies a mask on the entire image.
+
+The mask, its dimentions and area do not change often, so specializing the function for
```
Comment (serge-sans-paille) [inline README.md]: dimensions

Comment (serge-sans-paille) [reply README.md]: I'll do the change on my own, easier like that :-)

File: cpplib/include/easy/jit.h [hunk 5]
```diff
@@ -1,8 +1,8 @@
 #ifndef EASY
 #define EASY
 
-#include <easy/runtime/Context.h>
-#include <easy/runtime/Function.h>
+#include "../../../runtime/include/easy/runtime/Context.h"
```
Comment (serge-sans-paille) [inline cpplib/include/easy/jit.h]: This is... so wrong :-/ Why did you do that?

File: doc/readme/README.md.in [hunk 6]
```diff
@@ -0,0 +1,141 @@
+Easy::jit: A just-in-time compiler for C++
+==========================================
+
```
Comment (serge-sans-paille) [inline doc/readme/README.md.in]: shit, I missed that one, it probably meas I should have updated this one...

Comment (jmmartinez) [reply doc/readme/README.md.in]: I removed that, but I moved all the headers required by the user to the same directory, to catch them with only one "-I" flag.

**Comment by jmmartinez** [comment]
Now it's possible to build the example by specifying -DEASY_JIT_EXAMPLE=1 and it's correctly tested.

**Comment by serge-sans-paille** [comment]
@jmmartinez you probably need to rebase. LGTM otherwise.

**Comment by jmmartinez** [comment]
Rebased and Green!

**Pull request merged**